heron/common/src/java/com/twitter/heron/common/config/SystemConfig.java

```diff
@@ -59,6 +59,12 @@
       = "heron.metrics.export.interval.sec";
 
   /**
+   * The maximum number of exceptions in one MetricPublisherPublishMessage protobuf
+   */
+  public static final String HERON_METRICS_MAX_EXCEPTIONS_COUNT
+      = "heron.metrics.max.exceptions.count";
```

Comment:
This name on it's own doesn't clearly convey it's meaning. Can we change it to convey that it's the max exceptions in a protobuf? Something like `heron.metrics.max.exceptions.per.message.count` or something else more clear.